sql/testdata/builtin_function

```diff
@@ -332,3 +332,286 @@ SELECT extract(nanosecond from timestamp '2001-04-10 12:04:59.34565423')
 
 query error extract: unsupported timespan: nansecond
 SELECT extract(nansecond from timestamp '2001-04-10 12:04:59.34565423')
+
+query T
+SELECT abs(-1.0)
+----
+1
+
+query T
+SELECT abs(-0.0)
+----
+0
+
+query T
+SELECT abs(1.0)
```

Comment:
Can you add a float that is != to an int.

Also might be worth consolidating some of these tests to look like

SELECT abs(-1.0), abs(1.0), abs(-0.0), abs(0),  abs(1)